src/core-layers/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -130,7 +130,10 @@ export default class SolidPolygonLayer extends Layer {
 
   updateGeometry({props, oldProps, changeFlags}) {
     const geometryConfigChanged = props.extruded !== oldProps.extruded ||
-      props.wireframe !== oldProps.wireframe || props.fp64 !== oldProps.fp64;
+      props.wireframe !== oldProps.wireframe || props.fp64 !== oldProps.fp64 ||
+      (changeFlags.updateTriggersChanged && (
+        changeFlags.updateTriggersChanged.all ||
+        changeFlags.updateTriggersChanged.getPolygon));
 
     // check if updateTriggers.getElevation has been triggered
     const getElevationTriggered = changeFlags.updateTriggersChanged &&
```

Comment:
This check should be removed, and check `changeFlags.updateTriggersChanged.getElecation`

Reply:
But my code in lines 133-136 should be ok right?